common/backoff/jitter.go

```diff
@@ -35,6 +35,9 @@ func JitDuration(duration time.Duration, coefficient float64) time.Duration {
 // JitInt64 return random number from (1-coefficient)*input to (1+coefficient)*input, inclusive, exclusive
 func JitInt64(input int64, coefficient float64) int64 {
 	validateCoefficient(coefficient)
+	if coefficient == 0 {
+		return input
```

Comment:
let's cover the new lines in this file. there are existing tests already it should be straightforward to cover these